tck/src/main/java/org/reactivestreams/tck/PublisherVerification.java

```diff
@@ -128,7 +128,6 @@ public long maxElementsFromPublisher() {
 
   /**
    * Override and return {@code true} in order to skip executing tests marked as {@code Stochastic}.
-   * Such tests MAY sometimes fail even though the impl
```

Comment:
Or simply paraphrase the docs from tck/README:

`Stochastic tests can yield false positives ("be green") even if for some case, the given implementation may violate the tested behaviour. If for some reason, a stochastic test fails for your implementation, yet you are confident the implementation is correct you can disable them using this flag. Please contact the working group on https://github.com/reactive-streams/reactive-streams-jvm in case you encounter such issue with the tests.`